### ISSUE TITLE
>30 milestones not supported

### ISSUE THREAD

**Issue by justinmichaelvieira** [spyder-ide/loghub#100]
Hi loghub team; We are seeing an issue with having >30 milestones. After the 30th milestone is created, the 31st, 32nd etc are not picked up by loghub when trying to generate release notes. The response is `You didn't pass a valid milestone name!`, even though the milestone name passed is exactly what is registered on the github repo.

With milestone `v0.37.0` existing in our repo, I am seeing:

```(orchard-qtclient-KOPcPcJG) Justins-Air:~/orchard-qtclient$ loghub --milestone v0.37.0 -u angryrancor teamorchard/qtclient

LOGHUB: Querying issues for milestone Test

Password:
LOGHUB: You didn't pass a valid milestone name!
LOGHUB: The available milestones are: ['v0.4', '0.5.0', 'v0.6.0', 'v0.7.0', 'v0.8.0', 'v0.9.0', 'v0.10.0', 'v0.11.0', 'v0.12.0', 'v0.13.0', 'v0.14.0', 'v0.15.0', 'v0.16.0', 'v0.17.0', 'v0.21.0', 'v0.22.0', 'v0.23.0', 'v0.24.0', 'v0.25.0', 'v0.26.0', 'v0.27.0', 'v0.28.0', 'v0.29.0', 'v0.30.0', 'v0.31.0', 'v0.32.0', 'v0.33.0', 'v0.34.0', 'v0.35.0', 'v0.36.0-rc1']
```

Cause of this is almost definitely githubs rate limit settings: https://developer.github.com/v3/rate_limit/

It may be possible to query for results past 30 subsequently, if 30 milestones are brought back.

p.s. Thanks for creating and maintaining this great tool, we use it a lot and generally love it.

**Comment by ccordoba12** [comment]
Hey @justinmichaelvieira! Glad to know you keep using Loghub!

I think this problem was fixed in PR #99, which is part of our 0.4.0 version. So please update and let us know if it's fixed or not.

**Comment by goanpeca** [comment]
This is probably not valid anymore in terms that we get the latest first, however we do need to explore the limit.
